.github/docker-images/ubuntu-16-x64/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN curl -sSL https://github.com/Kitware/CMake/releases/download/v3.10.0/cmake-3
 # Clone and build Google Test
 ###############################################################################
 WORKDIR /tmp
-RUN wget https://github.com/google/googletest/archive/release-1.10.0.tar.gz \
+RUN wget https://github.com/google/googletest/archive/release-1.10.0.tar.gz --no-check-certificate \
```

Comment:
Ignoring the certificate chain opens us up to man in the middle (MITM) attacks on the software supply chain (eg dependencies used to build the software).  We should never do this. The likely reason for adding this flag is that wget is not finding the certificate store in the operating system. The solution is to install the package containing the certificates. Please add the line below `after` installing prereqs but `before` downloading dependencies.

```
RUN apt-get install -y ca-certificates
```

Reply:
Thanks for the heads up. I remember I did try doing that install but it still had issues verifying the cert. I haven't tested to see if the github workflow is able to complete yet, but hopefully I can find a fix for this asap.